composites/SnippetPreview/components/ProgressBar.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+const ProgressBarContainer = styled.progress`
+	box-sizing: border-box;
+	width: 100%;
+	height: 8px;
+	display: block;
+	margin-top: 5px;
+	border: none;
+	-webkit-appearance: none;
```

Comment:
Please don't use prefixed CSS properties, they should be added during the build process